core/src/apps/cardano/helpers/paths.py

```diff
@@ -11,6 +11,7 @@
 # fmt: off
 SCHEMA_PUBKEY = PathSchema("m/[44,1852]'/coin_type'/account'/*", SLIP44_ID)
 SCHEMA_ADDRESS = PathSchema("m/[44,1852]'/coin_type'/account'/[0,1,2]/address_index", SLIP44_ID)
+SCHEMA_ACCOUNT_LOOSE = PathSchema("m/[44,1852]'/coin_type'/**", SLIP44_ID)
```

Comment:
I would change `**` for `*'` - i.e. to enforce that the account index has to be hardened. It doesn't seem right to allow non-hardened account indices as that completely deviates from BIP-32/CIP-1852

Reply:
We discussed offline that in the end, it should be enough to just check for the length of the path to make the function safe - other validations happen elsewhere. Done in https://github.com/vacuumlabs/trezor-firmware/pull/45/commits/47e255e808bc814d3bfd5b374cdfa9ceffab780d.